frontend/tests/features/wheresMyGene/wheresMyGene.test.ts

```diff
@@ -302,36 +301,6 @@ describe("Where's My Gene", () => {
     );
   });
 
-  test("delete genes", async ({ page }) => {
```

Comment:
duplicate test that no longer works and was causing test failure due to testId changes